bin/setup

```diff
@@ -16,6 +16,7 @@ FileUtils.chdir APP_ROOT do
 
   puts "== Installing dependencies =="
   system("bundle check") || system!("bundle install")
+  system("bin/yarn check") || system!("bin/yarn install")
```

Comment:
The rails template actually has this if node support is enabled:
```ruby

  # Install JavaScript dependencies
  system("yarn install --check-files")
```
Not sure if the difference is meaningful though...